hummingbot/connector/exchange/bittrex/bittrex_exchange.py

```diff
@@ -0,0 +1,403 @@
+                retval.append(TradingRule(trading_pair,
+                                          min_order_size=Decimal(min_trade_size),
+                                          min_price_increment=Decimal(f"1e-{precision}"),
+                                          min_base_amount_increment=Decimal(f"1e-{precision}"),
+                                          ))
+            except KeyError:
+                self.logger().error(f"Trading-pair {market['symbol']} is not active. Skipping.")
+                continue
+            except Exception:
+                self.logger().error(f"Error parsing the trading pair rule {market}. Skipping.", exc_info=True)
+        return retval
+
+    async def _update_trading_fees(self):
+        resp = await self._api_get(
+            path_url=CONSTANTS.FEES_URL,
+            is_auth_required=True,
+        )
+        for fees in resp:
+            try:
+                trading_pair = await self.trading_pair_associated_to_exchange_symbol(symbol=fees["marketSymbol"])
+                self._trading_fees[trading_pair] = fees
+            except Exception:
+                continue
+
+    async def _all_trade_updates_for_order(self, order: InFlightOrder) -> List[TradeUpdate]:
+        trade_updates = []
+        if order.exchange_order_id is not None:
+            exchange_order_id = await order.get_exchange_order_id()
+            trades_from_exchange = await self._api_get(
+                path_url=CONSTANTS.ORDER_TRADES_URL.format(exchange_order_id),
+                is_auth_required=True,
+                limit_id=CONSTANTS.ORDER_TRADES_LIMIT_ID
+            )
+            for trade in trades_from_exchange:
+                percent_token = order.quote_asset
+                fee = TradeFeeBase.new_spot_fee(
+                    fee_schema=self.trade_fee_schema(),
+                    trade_type=order.trade_type,
+                    percent_token=percent_token,
+                    flat_fees=[TokenAmount(amount=Decimal(trade["commission"]), token=percent_token)]
+                )
+                trade_update = TradeUpdate(
+                    trade_id=str(trade["id"]),
+                    client_order_id=order.client_order_id,
+                    exchange_order_id=exchange_order_id,
+                    trading_pair=order.trading_pair,
+                    fee=fee,
+                    fill_base_amount=Decimal(trade["quantity"]),
+                    fill_quote_amount=Decimal(trade["quantity"]) * Decimal(trade["rate"]),
+                    fill_price=Decimal(trade["rate"]),
+                    fill_timestamp=_get_timestamp(trade["executedAt"]),
+                )
+                trade_updates.append(trade_update)
+
+        return trade_updates
+
+    async def _request_order_status(self, tracked_order: InFlightOrder) -> OrderUpdate:
+        exchange_order_id = await tracked_order.get_exchange_order_id()
+        params = {
+            "orderId": exchange_order_id
+        }
```

Comment:
According to the documentation this endpoint does not receive parameters (the order id is part of the URL).
https://bittrex.github.io/api/v3#operation--orders--orderId--get

Reply:
Removed the params